### ISSUE TITLE
Add content about our "culture" into Backdrop's messaging

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#288]
Some great feedback from @mtift @wesruv @quicksketch and @jackaponte about how Backdrop is more than just affordable software. Let's add some content and messaging about our "culture" (for lack of a better word) onto the website.

**Comment by jackaponte** [comment]
I'd assign this to me if I could figure out how to do so! Though of course others can & should feel free to participate, add thoughts, etc.

My next steps:

- [ ] review what's currently up on the site that reflects our organizational/project culture, draw out common themes
- [ ] create and share first draft of "Culture" page
- [ ] identify places on other pages where "Culture" page can be linked to or cited, or language can be expanded.

Aiming to move at least the first two items forward by the next community outreach meeting.

**Comment by mtift** [comment]
I'd be happy to help out with this. I'm not sure if this is what you want, but here are some questions that I have wondered about Backdrop:

* Does Backdrop stand for any principles, or is it simply a "comprehensive CMS"?
* Is it idealistic or realistic? That is to say, is it good enough right now or does it focus on a better Backdrop?
* Is everyone welcome in the Backdrop community or does it have ideal users/contributors?
* If I come for the Drupal-7 style code (which in actuality welcomes objects and classes), will I stay for the community?
* What is the "spirit of Backdrop"?
* Is Backdrop conservative, radical, or something else?
* Is it a means to an end or would it better be described as a human activity?
* Does/could/should/will Backdrop compete with [Drutopia](http://drutopia.org/)?

**Comment by klonos** [comment]
> I'd assign this to me if I could figure out how to do so!

@jackaponte that's because you aren't a member of the `/backdrop-ops/backdropcms.org` team. @jenlampton? @quicksketch? some help with that?

**Comment by klonos** [comment]
@mtift I wasn't aware of Drutopia. Thanx for the link, but I do not understand from the website what exactly that project is about. Will it be another Drupal fork, or is it a deviation/sub-group of the d.org community? How long has it been around for? Are they aware of our existence and do we meet the criteria of what motivated them to start in the first place? If so, then why not work together?

**Comment by mtift** [comment]
@klonos Drutopia is an new initiative within the Drupal project, just announced last week. I'm sure they know about Backdrop, but they would prefer to build tools with Drupal.

I brought it up because, far from just building a Drupal 8 distribution, the [Drupia team](http://www.drutopia.org/node/5) uses [politically-charged phrases](http://drutopia.org/drutopia-white-paper) like "grassroots groups," "the liberatory possibilities of free software," "people-centred economic models," and "progressive social change" that clearly identifies their approach (and their "culture"), and it's one that seems, to me at least, to be in stark contrast to the economic and technically-focused language on https://backdropcms.org as well as https://www.drupal.org/about.

I hope these comments are helpful.

**Comment by jackaponte** [comment]
Hi all--sorry to drop off here for so long! Dealing with some crises which have me fairly out of commission. I hope to respond to this thread soon and perhaps even have a first draft ready by our next Backdrop Outreach meeting, scheduled for September 22.

More soon!

**Comment by jackaponte** [comment]
Hi everyone--I'm sorry that I was never able to return to this work! Is the work outlined here still needed and important to the Backdrop community? If so, someone with more capacity than I may need to claim it in order for this to finally move forward!

**Comment by laryn** [comment]
Thanks, @jackaponte.
I do think something with a better sense of organizational culture would be good to have out there somewhere alongside the **affordability** aspect (lean, runs on shared hosting, backwards compatibility...), to @mtift's point. Some bullet point thoughts on some things that could potentially be (polished and) pulled out more explicitly in Backdrop's messaging:

- **people-centred**: I think the 80% rule and improved UX goals fit into this concept and the leadership structure shows a commitment for the project to be used by and led by diverse people rather than one company / subset / economic interest or another. 
- **invitational**, **educational**, **empowering**: as I understand it part of the goal is to provide a better product that works more simply for end users, but also to provide a welcoming on-ramp to new and potential developers. I've heard a number of comments about the welcoming feel of the community and the desire to grow that. The forums and the Gitter chat have a very educational thrust.
- **social good**: many of Backdrop's goals make it a perfect fit for nonprofits, its team structure show a commitment to diversity, and initiatives like Backdrop4Good show a commitment to help use Backdrop in the service of positive social change.

I'm sure there are more but these come to mind at the moment.

**Comment by jenlampton** [comment]
I also agree that having a statement of culture would be valuable to the community, and to potential new adopters as well.

@mtift I'd love to have you involved with helping us write this up, since @jackaponte seems to have a lot on their plate right now :) Here's a stab at your questions:

> Does Backdrop stand for any principles, or is it simply a "comprehensive CMS"?

@laryn's points above are all great ones, and I'd love to see those included.  I'd also like to point to our statements under [Audience](https://backdropcms.org/philosophy) about which members of our community we put first -- those with the highest needs. Those who "can do for themselves" come lower down than those who cannot.

> Is it idealistic or realistic? That is to say, is it good enough right now or does it focus on a better Backdrop?

Well, It's both. Of course we always want to make a better Backdrop, but we're trying to be honest and realistic about the problems that are facing our users **today**. We aim to solve today's problems before tomorrow's (for example, we fixing bugs in the current version of the software, before the fixing them in the version that's not released yet -- this differs significantly from "the Drupal way")

> Is everyone welcome in the Backdrop community or does it have ideal users/contributors?

Everyone is welcome, but we do have a specific audience in mind, and their needs will come first. If you are running a huge enterprise software-as-a-service platform on Backdrop -- that's great! -- but your PRs in the core queue will not be merged if the benefit doesn't apply to everyone* else as well.

> If I come for the Drupal-7 style code (which in actuality welcomes objects and classes), will I stay for the community?

Yes! We hope so anyway :)

> What is the "spirit of Backdrop"?

I don't know what this questions is asking. 

> Is Backdrop conservative, radical, or something else?

I'd say it's conservative because our decisions are carefully weighed and changes are made slowly. We aim to serve a very large user-base, but one that doesn't have deep pockets so our changes need to be within the scope of what they can afford (both financially, and emotionally!)

> Is it a means to an end or would it better be described as a human activity?

For most, it will be a means to an end. For the most committed of us it will be an activity :)

> Does/could/should/will Backdrop compete with Drutopia?

Backdrop and Drutopia serve the same audience, so it's likely they will compete with one another. However, both teams are on the same page, and we've been talking and working with the folks at Drutopia! We'd both like to determine some factors that will help potential adopters choose between the two. It's a little hard to pin down right now since Drutopia is still so new, but we're all on the same team :)
